<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerMetadata</class>
 <widget class="QDockWidget" name="LayerMetadata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layer Metadata</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QPushButton" name="pbRefresh">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(161, 226, 188);</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>tip: Ctrl-C to copy value to clipboard</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Layers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="itemsExpandable">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>false</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="headerCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="headerMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Group by</string>
           </property>
           <property name="textAlignment">
            <set>AlignLeft|AlignTop</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
           <property name="textAlignment">
            <set>AlignLeft|AlignTop</set>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QTableWidget" name="tabShow">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>42</y>
          <width>570</width>
          <height>326</height>
         </rect>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>2</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>Enable</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Property</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Applicable</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>12</y>
          <width>462</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Which metadata do you want to see?</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>plugin by Steven Kay (@stevefaeembra).

This is an alternative Layers panel which is designed to make it easier to get at Layer Metadata. If you spend a lot of time going into layer properties to extract things like extents and proj4 strings, this tool should make it easier and quicker.

You can choose how to group your layers using the combo box at the top.

It's not smart enough to detect changes in the layers table; you can force a refresh with the big green button below, or change the grouping value.

You can copy any value to the clipboard using Ctrl-C.

You can also customise the plugin in the settings tab.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="cbGrouping">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>class</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>crs</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>extent</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>geom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>id</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>layername</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>provider</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>url</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Group layers by...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
